lib/helpers.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Phoenix-RTOS
```

Comment:
Nit: two things missing in this header (in the kernel too, missed that):
- Operating system loader
- This file is part of Phoenix-RTOS.

(see other files)

Reply:
Done